packages/x-charts-base/src/FakeCss/index.ts

```diff
@@ -0,0 +1,49 @@
+import { styled } from '@mui/material';
+
+export const FakeCss = styled('div')(({ theme }) => ({
+  '& .ChartsSurface-root': {
+    height: 'var(--chart-surface-height, 100%)',
+    width: 'var(--chart-surface-width, 100%)',
+    display: 'flex',
+    position: 'relative',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    overflow: 'hidden',
+    /* This prevents default touch actions when using the svg on mobile devices.
+       For example, prevent page scroll & zoom. */
+    touchAction: 'pan-y',
+    userSelect: 'none',
+    gridArea: 'chart',
```

Comment:
If these are functional styles that we should always have they should be directly set on the element via the style tag. If not, it's enough to have them in a demo.